### PR TITLE
add BetaFPV Lite, GEPRC Nano, iFlight to Receiver Selection

### PR DESCRIPTION
**Pull request by deadbytefpv**

### CODE REVIEW

File: docs/hardware/receiver-selection.md [hunk 1]
```diff
@@ -13,14 +13,17 @@ ExpressLRS is hashtag blessed with the benefit of there being many receivers ava
 | Jumper AION Mini RX | :moneybag: | :straight_ruler::straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3: | :checkered_flag:|
 | Vantac RX | :moneybag: | :straight_ruler::straight_ruler: | :thumbsup_tone3: | 20dBm | :whale::whale: | :thumbsup_tone3: | :checkered_flag:|
 | BetaFPV Nano 2.4 | :moneybag::moneybag: | :straight_ruler::straight_ruler: | :thumbsup_tone3:  | 20dBm | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |
+| BetaFPV Lite 2.4 | :moneybag::moneybag: | :straight_ruler: | :thumbsup_tone3:  | :thumbsdown_tone3: | :whale: | :thumbsup_tone3:  | :checkered_flag: |
```
Comment (SunjunKim) [inline docs/hardware/receiver-selection.md:16]: This has no LNA

| BetaFPV Lite 2.4 | :moneybag::moneybag: | :straight_ruler: | :thumbsdown_tone3:  | :thumbsdown_tone3: | :whale: | :thumbsup_tone3:  | :checkered_flag: |

File: docs/hardware/receiver-selection.md [hunk 2]
```diff
@@ -13,14 +13,17 @@ ExpressLRS is hashtag blessed with the benefit of there being many receivers ava
...
 | Ghost ATTO | :moneybag::moneybag::moneybag::moneybag: | :straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsdown_tone3: | :thumbsdown_tone3: |
+| GEPRC Nano | :moneybag: | :straight_ruler::straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |
 | HappyModel PP | :moneybag::moneybag: | :straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale: | :thumbsdown_tone3: | :checkered_flag: |
 | HappyModel EP1 | :moneybag: | :straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |
 | HappyModel EP2 | :moneybag: | :straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale: | :thumbsup_tone3:  | :checkered_flag: |
 | HGLRC 2400RX | :moneybag::moneybag: | :straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: | 
+| iFlight ELRS | :moneybag: | :straight_ruler::straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |
```
Comment (SunjunKim) [inline docs/hardware/receiver-selection.md:26]: It has LNA/PA equipped - 

| iFlight ELRS | :moneybag: | :straight_ruler::straight_ruler: | :thumbsup_tone3: | 20dBm | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |

File: docs/hardware/receiver-selection.md [hunk 3]
```diff
@@ -39,6 +42,7 @@ ExpressLRS is hashtag blessed with the benefit of there being many receivers ava
 | HappyModel ES900RX | :moneybag::moneybag: | :straight_ruler: | :thumbsup_tone3:  | :thumbsup_tone3:  | :whale::whale: | :thumbsup_tone3:  | :thumbsup_tone3:  |
 | Jumper R9 Mini | :moneybag::moneybag: | :straight_ruler::straight_ruler: | :thumbsup_tone3:  | :thumbsup_tone3:  | :whale::whale: | :thumbsdown_tone3: | :thumbsdown_tone3: |
 | NamimnoRC Voyager | :moneybag::moneybag::moneybag: | :straight_ruler::straight_ruler: | :thumbsup_tone3:  | :thumbsup_tone3:  | :whale::whale: | :thumbsdown_tone3: | :thumbsup_tone3:  |
+| GEPRC Nano | :moneybag::moneybag: | :straight_ruler::straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |
```
Comment (SunjunKim) [inline docs/hardware/receiver-selection.md:45]: Isn't this a duplicated entry w. the Line 21?
Also there are two: NanoSE and Nano

| GEPRC NanoSE | :moneybag::moneybag: | :straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale: | :thumbsup_tone3:  | :checkered_flag: |
| GEPRC Nano | :moneybag::moneybag: | :straight_ruler::straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |

**Review by SunjunKim — CHANGES_REQUESTED** [review]
Few changes are required

File: docs/hardware/receiver-selection.md (hunk 1 above)
Comment (deadbytefpv) [reply docs/hardware/receiver-selection.md:16]: which is why there's a thumbsdown?

File: docs/hardware/receiver-selection.md (hunk 3 above)
Comment (deadbytefpv) [reply docs/hardware/receiver-selection.md:45]: Internally, there's only a Nano, but I did thought of putting in one for the SE

this section is for the 900MHz ones

File: docs/hardware/receiver-selection.md (hunk 2 above)
Comment (deadbytefpv) [reply docs/hardware/receiver-selection.md]: I have updated this line.

File: docs/hardware/receiver-selection.md (hunk 1 above)
Comment (SunjunKim) [reply docs/hardware/receiver-selection.md:16]: You've got thumbsup in the LNA column

**Pull request merged**